librocksdb_sys/libtitan_sys/Cargo.toml

```diff
@@ -7,7 +7,7 @@ links = "titan"
 [dependencies]
 bzip2-sys = "0.1.8+1.0.8"
 libc = "0.2.11"
-libz-sys = { version = "1.0.25", features = ["static"] }
+libz-sys = { version = "1.1.2", features = ["static"] }
```

Comment:
shall we just specify "1.1"?

Reply:
Sure! `1.1` is ok because `1.1.2` was released.